From 54cebbd0de80a271a9f9f49e0affc905defe34f6 Mon Sep 17 00:00:00 2001
From: Anton Lavrik <alavrik@piqi.org>
Date: Sun, 7 Apr 2013 02:07:58 -0500
Subject: [PATCH 3/9] Move installing of piqic "make install" to "make
 ocaml-install"

... because "piqic" is now ocaml-specific (and soon to be renamed piqic-ocaml)
---
 Makefile |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 7a7be52..95de550 100644
--- a/Makefile
+++ b/Makefile
@@ -50,7 +50,6 @@ build-dir:
 install:
 	-install -d $(DESTDIR)$(PIQI_PREFIX)/bin
 	install piqi-tools/piqi $(DESTDIR)$(PIQI_PREFIX)/bin
-	install piqic/piqic $(DESTDIR)$(PIQI_PREFIX)/bin
 
 
 ocaml:
@@ -61,6 +60,8 @@ ocaml-install: ocaml-uninstall
 	test -d $(PIQI_OCAML_DESTDIR) || mkdir -p $(PIQI_OCAML_DESTDIR)
 	ocamlfind install piqi `ls $(PIQI_BUILD)/lib/ocaml/piqi/*`
 	$(MAKE) -C deps install
+	-install -d $(DESTDIR)$(PIQI_PREFIX)/bin
+	install piqic/piqic $(DESTDIR)$(PIQI_PREFIX)/bin
 
 
 ocaml-uninstall:
-- 
1.7.9.5

